zerver/management/commands/bulk_change_user_name.py

```diff
@@ -7,25 +7,32 @@
 from django.core.management.base import BaseCommand
 
 from zerver.lib.actions import do_change_full_name
-from zerver.models import UserProfile, get_user_profile_by_email
+from zerver.models import UserProfile, get_realm, get_user_for_mgmt
 
 class Command(BaseCommand):
     help = """Change the names for many users."""
 
     def add_arguments(self, parser):
         # type: (ArgumentParser) -> None
+        parser.add_argument(
+            '-r', '--realm', nargs='?', default=None,
+            dest='string_id',
+            type=str,
+            help='The name of the realm in which you are bulk changing user names.')
+
         parser.add_argument('data_file', metavar='<data file>', type=str,
                             help="file containing rows of the form <email>,<desired name>")
 
     def handle(self, *args, **options):
         # type: (*Any, **str) -> None
         data_file = options['data_file']
+        realm = get_realm(options["string_id"])
```

Comment:
We should do the same check for realm here as `./manage.py bulk_change_user_name test.txt  --realm invalidrealm' still works where content of test.txt is 

```
iago@zulip.com,iago2
```